designs/cilium-configuration-policy.md

```diff
@@ -0,0 +1,119 @@
+# Configure Cilium CNI
+
+## Introduction
+
+**Problem:** The Cilium CNI plugin (built-in default for EKS Anywhere) supports defining and enforcement of network policies. Network policies determine how pods within a cluster are able to communicate with one another. Cilium accepts policy enforcement modes from the users to determine the allowed traffic between endpoints. Currently there is no way to configure Cilium launched by EKS Anywhere to use any mode apart from the default mode, which allows all traffic.
+
+### Goals and Objectives
+
+As an EKS Anywhere user:
+
+* I want to have the ability to configure Cilium's [supported policy enforcement modes](https://docs.cilium.io/en/v1.8/policy/intro/).
```

Comment:
nit: should we reference v1.9 or v1.10 or latest (least preferred) here?
```suggestion
* I want to have the ability to configure Cilium's [supported policy enforcement modes](https://docs.cilium.io/en/v1.9/policy/intro/).
```

Reply:
I think either of the versions should be fine since they all contain policy enforcement modes